portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,43 @@
 
 package require json::write
 
+proc format_string {string} {
+	if {[llength $string] > 1} {
+		foreach element $string {
+			lappend new_string "\"$element\""
+		}
+	} else {
+		set new_string [list "\"$string\""]
```

Comment:
In what situation does the above foreach loop not work on a list of length 1?

Reply:
I was getting error in case of "depends_lib" and "depends_build". But now I realise it was because in several cases the values of "depends_lib" and "depends_build" are empty. There sure is a better way to go through this. What do you suggest?